src/lib/mina_base/user_command.ml

```diff
@@ -400,6 +408,8 @@ module Well_formedness_error = struct
         "Zero vesting period"
     | Zkapp_too_big err ->
         sprintf "Zkapp too big (%s)" (Error.to_string_hum err)
+    | Transaction_type_disabled ->
+        "Zkapps are disabled"
```

Comment:
Now that we changed the name of an error, we should probably update description too